tool/gravity/cli/clusterupdate.go

```diff
@@ -213,6 +213,17 @@ func executeUpdatePhase(env *localenv.LocalEnvironment, environ LocalEnvironment
 	if operation.Type != ops.OperationUpdate {
 		return trace.NotFound("no active update operation found")
 	}
+
+	statusList, err := collectAgentStatus(env)
```

Comment:
So this will also be executed not just for resume, but for any individual phase execution as well (i.e. when you do `gravity plan execute --phase=xxx`), which is probably not what we want. If you look at how resume is invoked, it sets the `PhaseID` to `/` which indicates it is resume being requested: https://github.com/gravitational/gravity/blob/version/5.5.x/tool/gravity/cli/run.go#L430. I would make sure that this agents check is only executed for `gravity plan resume` and not for `gravity plan execute --phase=`.